server/game/stateWatchers/CardsDrawnThisPhaseWatcher.ts

```diff
@@ -3,10 +3,12 @@ import { StateWatcherName } from '../core/Constants';
 import { StateWatcherRegistrar } from '../core/stateWatcher/StateWatcherRegistrar';
 import Player from '../core/Player';
 import { Card } from '../core/card/Card';
+import { InPlayCard } from '../core/card/baseClasses/InPlayCard';
+import * as Contract from '../core/utils/Contract';
 
 export interface DrawnCardEntry {
     player: Player;
-    amount: number;
+    card: InPlayCard;
```

Comment:
`InPlayCard` doesn't include events, right now there isn't a good type for this due to some issues with the type hierarchy so it's probably easiest to just do `Card`.